spec/requests/registration_spec.rb

```diff
@@ -0,0 +1,51 @@
+require "rails_helper"
+
+RSpec.describe "Registration", :type => :request do
```

Comment:
Prefer `type: :request` over :type => :request

And we can follow controller path in the route + the action name, like the following:
spec/api/v1/registration/create_spec.rb

And change spec line for this line to be consistent
`describe 'POST api/v1/registration', type: :request do`

WDYT?